templates/community.html

```diff
@@ -114,5 +114,24 @@
             </div>
         </div>
     </a>
+    <a class="card" href="/community/people/">
```

Comment:
Not something we need to do this PR, but I think we should consider merging `community` and `people` into one page. Community links at the top (smaller / more consolidated than the current list) directly followed by the people list.

It would give the people page more prominence / discoverability. And it would give more of an impression of "community driven development".